skeletons/oer_support.c

```diff
@@ -35,7 +35,7 @@ oer_fetch_length(const void *bufptr, size_t size, size_t *len_r) {
     }
 
     len_len = (first_byte & 0x7f);
-    if((1 + len_len) > size) {
+    if((size_t)(1 + len_len) > size) {
```

Comment:
Why this is necessary? len_len is already unsigned.

Reply:
Here is the travis log https://travis-ci.org/velichkov/asn1c/jobs/289628105#L1138-L1140
>   CC       oer_support.lo
oer_support.c: In function ‘oer_fetch_length’:
oer_support.c:38:22: error: comparison between signed and unsigned integer expressions [-Werror=s

Reply:
Oh. You could just switch uint8_t to size_t as len_len's type, then. Avoids confusion.